core/services/ocr2/plugins/ocr2keeper/evm20/registry.go

```diff
@@ -140,7 +142,7 @@ type EvmRegistry struct {
 	poller        logpoller.LogPoller
 	addr          common.Address
 	client        client.Client
-	registry      Registry
+	Registry      Registry
```

Comment:
no need to make this uppercase

Reply:
how can i get around this unexported issue?

<img width="534" alt="Screenshot 2023-08-10 at 1 02 29 PM" src="https://github.com/smartcontractkit/chainlink/assets/82334955/ba7f44a1-7730-4e7a-9bcb-020477342ccd">
<img width="713" alt="Screenshot 2023-08-10 at 1 01 17 PM" src="https://github.com/smartcontractkit/chainlink/assets/82334955/7caee812-f491-4b0d-aa73-18c4f6b5c09d">
<img width="997" alt="Screenshot 2023-08-10 at 1 01 34 PM" src="https://github.com/smartcontractkit/chainlink/assets/82334955/fafe7f70-7f54-412f-90b8-560ab3b2ac83">

Reply:
let's keep this internal to registry.
we can create a new registry wrapper following how this is created, basically sth like:
```registry, err := keeper_registry_wrapper2_0.NewKeeperRegistry(addr, chain.Client())
		if err != nil {
			return nil, fmt.Errorf("%w: failed to create caller for address and backend", ErrInitializationFailure)
		}```